### DISCUSSION TITLE
【第四期飞桨黑客松】模型套件开源贡献——遥感影像智能解译开发套件

### DISCUSSION THREAD

**Discussion by cloud2009** [PaddlePaddle/Paddle#50732] [General]
[PaddlePaddle Hackathon 第四期](https://github.com/PaddlePaddle/Paddle/issues/50629#rs)
大家可以在这里讨论有关遥感影像智能解译开发套件任务的问题哦～

**Comment by Harryoung** [comment]
151号题目的数据在这里~
链接: https://pan.baidu.com/s/1BXoQX6Z4KONbkm1gw6BXNw?pwd=ye92 提取码: ye92
